content/docs/getting-started/installing.md

```diff
@@ -97,3 +100,41 @@ To reload the extension when testing, click the "Reload" button on the extension
   The unexpected WebExtension manifest property warnings may safely be ignored, since they are required by Chrome.
 {{< /admonition >}}
 
+### Installing on Firefox for Android
+
+{{< admonition info >}}
+  This is only recommended if there is a mobile specific issue that cannot be easily replicated with the browser's developer tools since extensions loaded this way are temporary and must be reinstalled over USB every time the app is restarted.
+{{< /admonition >}}
+
+#### One time setup
+
+##### Desktop
+
+1. Download and extract the Android SDK Platform Tools ([Windows](https://dl.google.com/android/repository/platform-tools-latest-windows.zip), [MacOS](https://dl.google.com/android/repository/platform-tools-latest-darwin.zip), [Linux](https://dl.google.com/android/repository/platform-tools-latest-linux.zip)).
+1. Add the folder to the PATH environment variable so it can be used anywhere.
+1. Install web-ext with npm: `npm install --global web-ext`.
+
+##### Android
+
+1. In the Android settings app, open the about page and tap the build number 7 times.
+1. Navigate to the "Developer options" page and enable "USB debugging".
+1. Open the Firefox app and enable "Remote debugging via USB" in its settings.

```

Comment:
We could clarify this guide does not consider Firefox Beta nor Nightly channels.

```suggestion
1. Install the standard Firefox browser (not beta nor nightly channels) from the Play Store.
1. Open the Firefox app and enable "Remote debugging via USB" in its settings.
```